src/crimson/os/seastore/segment_manager.cc

```diff
@@ -71,7 +71,7 @@ SegmentManager::get_segment_manager(
 	  });
       }).then([&](auto ret) -> crimson::os::seastore::SegmentManagerRef {
 	crimson::os::seastore::SegmentManagerRef sm;
-	logger().error("NR_ZONES: {}", nr_zones);
+	logger().info("Found {} zones.", nr_zones);
```

Comment:
Let's convert the logging in these files to use the macros in logging.h to match the rest of seastore.

Reply:
Ok, I have changed this, will update the PR.